openhands/core/message.py

```diff
@@ -34,14 +37,14 @@ def serialize_model(self):
 
 
 class ImageContent(Content):
-    type: ContentType = ContentType.IMAGE_URL
+    type: str = ContentType.IMAGE_URL.value
```

Comment:
Any reason for changing the Enum type to string?

Reply:
The `type` was ending up as some cryptic type naming format in the message, but I don't have that log in front of me anymore. It wasn't only just "text" or "image_url", but ContentType<TEXT>.... something, iirc